### ISSUE TITLE
Can't get `getType<T>()` to work

### ISSUE THREAD

**Issue by 3p3r** [Hookyns/tst-reflect#77]
Hello! Awesome project. Thank you!

I have a question, not sure if I am hitting a setup bug, but I am using tst-reflect through webpack and ttypescipt. I have this snippet:

```TS
interface IFoo {}
class Foo implements IFoo {}

const foo = new Foo();
getType<Foo>();
getType<typeof foo>();
getType<IFoo>();
```

I can see my source is instrumented with reflection data correctly, but these calls are all compiling to empty `getType()` calls. Am I doing something wrong?

**Comment by 3p3r** [comment]
If it helps, I am using `ttsc.transpileModule` directly in code.

**Comment by Hookyns** [comment]
Hi @3p3r,
here is an StackBlitz [example](https://stackblitz.com/edit/tst-reflect-webpack?file=index.ts) with regular Webpack setup that works fine.

Can you please post a repro with your usage of `ttsc.transpileModule`?

**Comment by 3p3r** [comment]
Thanks for the quick response! So I am writing a new webpack loader for my project that happens to use tst-reflect in it. I am using the following snippet:

```TS
import ttsc from "ttypescript";
const compiledTSX = `...`; // some source
console.log(ttsc.transpileModule(compiledTSX, {
    compilerOptions: {
      configFilePath: path.resolve(process.cwd(), "tsconfig.json"),
      target: ttsc.ScriptTarget.ESNext,
      jsx: ttsc.JsxEmit.ReactJSX,
      jsxImportSource: "jsx",
      experimentalDecorators: true,
      module: ttsc.ModuleKind.NodeNext,
      allowJs: true,
      esModuleInterop: true,
      alwaysStrict: true,
      skipDefaultLibCheck: true,
      skipLibCheck: true,
      typeRoots: ["node_modules/@types"],
      // @ts-expect-error
      plugins: [{ transform: require.resolve("tst-reflect-transformer") }],
    },
  }).outputText)
```
The `tsconfig.json` file on disk is just an empty file. Is this a supported use-case?

**Comment by Hookyns** [comment]
@3p3r 

I played with it a little...

The issue is in transpileModule inside typescript itself. It does not load declarations of modules etc. It only transpile given module.
`getType<>()` calls were not recognized as our getType calls so we ignored them. Recognition is based on declaration.

I did some changes, so every `getType<T>()` without declaration (typechecker returns error for such function), called exactly with one type argument, will be considered as our.
Here is new [demo](https://stackblitz.com/edit/tst-reflect-transpile-module?file=index.ts).

Included in `tst-reflect-transformer@0.12.5`.
